src/com/projectkorra/projectkorra/airbending/flight/FlightMultiAbility.java

```diff
@@ -227,14 +228,15 @@ public void run() {
 						if (!GeneralMethods.isRegionProtectedFromBuild(this.player, e.getLocation())) {
 							final LivingEntity le = (LivingEntity) e;
 							DamageHandler.damageEntity(le, this.speed / 2, this);
-							le.setVelocity(this.player.getVelocity().clone().multiply(2 / 3));
+							GeneralMethods.setVelocity(this,le, this.player.getVelocity().clone().multiply(2 / 3));
```

Comment:
Please change to `GeneralMethods.setVelocity(this, le, this.player.getVelocity().clone().multiply(2 / 3));`